server/src/main/scala/actors/PortStateActor.scala

```diff
@@ -22,27 +24,32 @@ import scala.language.postfixOps
 
 
 object PortStateActor {
-  def apply(now: () => SDateLike, liveCrunchStateActor: ActorRef, forecastCrunchStateActor: ActorRef)
+  def apply(now: () => SDateLike,
+            liveCrunchStateActor: ActorRef,
+            forecastCrunchStateActor: ActorRef,
+            queues: Map[Terminal, Seq[Queue]])
```

Comment:
it would make sense to make the variable names here the same as the Actor constructor.

Reply:
:+1: